apps/vr-tests-react-components/package.json

```diff
@@ -46,7 +46,7 @@
     "@fluentui/react-radio": "^9.1.1",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27150&commitId=67cd959ac0a9f33485d2bb807559c8fef651797f&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1891 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+inactive+badge.normal.chromium.png|1| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>